### PR TITLE
Fix/input tel format options

### PR DESCRIPTION
**Pull request by gerjanvangeest**
## What I did

1. Substract and export getFlagSymbol function
2. Add option to style the country-code with parentheses in the formatter
3. Remove unwanted characters in input-tel parser

### CODE REVIEW

File: packages/input-tel/src/parsers.js [hunk 1]
```diff
@@ -18,10 +18,11 @@ export function parsePhoneNumber(viewValue, { regionCode }) {
 
   // eslint-disable-next-line prefer-destructuring
   const PhoneNumber = /** @type {AwesomePhoneNumber} */ (PhoneUtilManager.PhoneUtil);
-
+  const regex = /[+0-9]+/gi;
```
Comment (tlouisse) [inline packages/input-tel/src/parsers.js:21]: Nice.

Only having a quick look atm, but do we still catch all cases, like:

pn.getNumber( 'national' );        // -> '070-712 34 56'
pn.getNumber( 'rfc3966' );         // -> 'tel:+46-70-712-34-56'

File: packages/input-tel/test/preprocessors.test.js [hunk 2]
```diff
@@ -29,4 +29,54 @@ describe('liveFormatPhoneNumber', () => {
       }),
     ).to.eql({ caretIndex: 12, viewValue: '+31 6 12345678' });
   });
+
+  describe('with formatCountryCodeStyle is set to parantheses', () => {
```
Comment (tlouisse) [inline packages/input-tel/test/preprocessors.test.js:33]:
```suggestion
  describe('with formatCountryCodeStyle is set to parentheses', () => {
```

File: packages/input-tel/src/parsers.js (hunk 1 above)
Comment (gerjanvangeest) [reply packages/input-tel/src/parsers.js:21]: I added the following tests:

it('removes unwanted characters', () => {
    expect(parsePhoneNumber('(+31)707123456', { regionCode: 'NL' })).to.equal('+31707123456');
    expect(parsePhoneNumber('+31 70 7123456', { regionCode: 'NL' })).to.equal('+31707123456');
    expect(parsePhoneNumber('+31-70-7123456', { regionCode: 'NL' })).to.equal('+31707123456');
    expect(parsePhoneNumber('+31|70|7123456', { regionCode: 'NL' })).to.equal('+31707123456');
  });
  
  Could add one extra for `rfc3966`

Comment (tlouisse) [reply packages/input-tel/src/parsers.js:21]: Most likely catches it like this, but would be nice to have the test for rfc3966 indeed

**Review by tlouisse — APPROVED** [review]

**Pull request merged**